### ISSUE TITLE
com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Table 'chaosblade.t_chaos_m_quartz_simple_triggers' doesn't exist

### ISSUE THREAD

**Issue by tytyty123** [chaosblade-io/chaosblade-box#91]
chaosblade-box-1.0.1.jar启动报错，表不存在，在chaosblade-box-ddl.sql里也没找到chaosblade.t_chaos_m_quartz_simple_triggers建表语句

2022-06-07 15:58:36.233 ERROR 31603 --- [           main] c.a.c.b.d.s.BaseSchedulerService         : add scheduler failed,jobId:1534079843653369857

org.quartz.JobPersistenceException: Couldn't remove trigger: Table 'chaosblade.t_chaos_m_quartz_simple_triggers' doesn't exist
        at org.quartz.impl.jdbcjobstore.JobStoreSupport.removeTrigger(JobStoreSupport.java:1462) ~[quartz-2.3.0.jar!/:na]
        at org.quartz.impl.jdbcjobstore.JobStoreSupport$10.execute(JobStoreSupport.java:1432) ~[quartz-2.3.0.jar!/:na]
        at org.quartz.impl.jdbcjobstore.JobStoreCMT.executeInLock(JobStoreCMT.java:245) ~[quartz-2.3.0.jar!/:na]
        at org.quartz.impl.jdbcjobstore.JobStoreSupport.removeTrigger(JobStoreSupport.java:1428) ~[quartz-2.3.0.jar!/:na]
        at org.quartz.core.QuartzScheduler.unscheduleJob(QuartzScheduler.java:1059) ~[quartz-2.3.0.jar!/:na]
        at org.quartz.core.QuartzScheduler.deleteJob(QuartzScheduler.java:960) ~[quartz-2.3.0.jar!/:na]
        at org.quartz.impl.StdScheduler.deleteJob(StdScheduler.java:301) ~[quartz-2.3.0.jar!/:na]
        at com.alibaba.chaosblade.box.dao.scheduler.quartz.QuartzSchedulerService.rescheduleCronJob(QuartzSchedulerService.java:81) ~[chaosblade-box-dao-1.0.1.jar!/:na]
        at com.alibaba.chaosblade.box.dao.scheduler.BaseSchedulerService.addSchedulerJob(BaseSchedulerService.java:41) ~[chaosblade-box-dao-1.0.1.jar!/:na]
        at com.alibaba.chaosblade.box.dao.scheduler.job.ActivityAsyncCheckSchedulerJob.afterPropertiesSet(ActivityAsyncCheckSchedulerJob.java:62) [chaosblade-box-dao-1.0.1.jar!/:na]
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeInitMethods(AbstractAutowireCapableBeanFactory.java:1687) [spring-beans-4.3.13.RELEASE.jar!/:4.3.13.RELEASE]
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1624) [spring-beans-4.3.13.RELEASE.jar!/:4.3.
        at org.quartz.impl.jdbcjobstore.JobStoreSupport$10.execute(JobStoreSupport.java:1432) ~[quartz-2.3.0.jar!/:na]
        at org.quartz.impl.jdbcjobstore.JobStoreCMT.executeInLock(JobStoreCMT.java:245) ~[quartz-2.3.0.jar!/:na]
        at org.quartz.impl.jdbcjobstore.JobStoreSupport.removeTrigger(JobStoreSupport.java:1428) ~[quartz-2.3.0.jar!/:na]
        at org.quartz.core.QuartzScheduler.unscheduleJob(QuartzScheduler.java:1059) ~[quartz-2.3.0.jar!/:na]
        at org.quartz.core.QuartzScheduler.deleteJob(QuartzScheduler.java:960) ~[quartz-2.3.0.jar!/:na]
        at org.quartz.impl.StdScheduler.deleteJob(StdScheduler.java:301) ~[quartz-2.3.0.jar!/:na]
        at com.alibaba.chaosblade.box.dao.scheduler.quartz.QuartzSchedulerService.rescheduleCronJob(QuartzSchedulerService.java:81) ~[chaosblade-box-dao-1.0.1.jar!/:na]
        at com.alibaba.chaosblade.box.dao.scheduler.BaseSchedulerService.addSchedulerJob(BaseSchedulerService.java:41) ~[chaosblade-box-dao-1.0.1.jar!/:na]
        at com.alibaba.chaosblade.box.dao.scheduler.job.ActivityAsyncCheckSchedulerJob.afterPropertiesSet(ActivityAsyncCheckSchedulerJob.java:62) [chaosblade-box-dao-1.0.1.jar!/:na]
        at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:306) [spring-beans-4.3.13.RELEASE.jar!/:4.3.13.RELEASE]
        at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:230) [spring-beans-4.3.13.RELEASE.jar!/:4.3.13.RELEASE]
        at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:302) [spring-beans-4.3.13.RELEASE.jar!/:4.3.13.RELEASE]
        at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:197) [spring-beans-4.3.13.RELEASE.jar!/:4.3.13.RELEASE]
        at com.zaxxer.hikari.pool.HikariProxyPreparedStatement.executeUpdate(HikariProxyPreparedStatement.java) ~[HikariCP-2.5.1.jar!/:na]
        at org.quartz.impl.jdbcjobstore.SimpleTriggerPersistenceDelegate.deleteExtendedTriggerProperties(SimpleTriggerPersistenceDelegate.java:58) ~[quartz-2.3.0.jar!/:na]
        at org.quartz.impl.jdbcjobstore.StdJDBCDelegate.deleteTriggerExtension(StdJDBCDelegate.java:1577) ~[quartz-2.3.0.jar!/:na]
        at org.quartz.impl.jdbcjobstore.StdJDBCDelegate.deleteTrigger(StdJDBCDelegate.java:1561) ~[quartz-2.3.0.jar!/:na]
2022-06-07 15:58:36.233 ERROR 31603 --- [           main] c.a.c.b.d.s.BaseSchedulerService         : add scheduler failed,jobId:1534079843653369857

org.quartz.JobPersistenceException: Couldn't remove trigger: Table 'chaosblade.t_chaos_m_quartz_simple_triggers' doesn't exist
        at org.quartz.impl.jdbcjobstore.JobStoreSupport.removeTrigger(JobStoreSupport.java:1462) ~[quartz-2.3.0.jar!/:na]
        at org.quartz.impl.jdbcjobstore.JobStoreSupport$10.execute(JobStoreSupport.java:1432) ~[quartz-2.3.0.jar!/:na]
        at org.quartz.impl.jdbcjobstore.JobStoreCMT.executeInLock(JobStoreCMT.java:245) ~[quartz-2.3.0.jar!/:na]
        at org.quartz.impl.jdbcjobstore.JobStoreSupport.removeTrigger(JobStoreSupport.java:1428) ~[quartz-2.3.0.jar!/:na]
        at org.quartz.core.QuartzScheduler.unscheduleJob(QuartzScheduler.java:1059) ~[quartz-2.3.0.jar!/:na]
        at org.quartz.core.QuartzScheduler.deleteJob(QuartzScheduler.java:960) ~[quartz-2.3.0.jar!/:na]
        at org.quartz.impl.StdScheduler.deleteJob(StdScheduler.java:301) ~[quartz-2.3.0.jar!/:na]
        at com.alibaba.chaosblade.box.dao.scheduler.quartz.QuartzSchedulerService.rescheduleCronJob(QuartzSchedulerService.java:81) ~[chaosblade-box-dao-1.0.1.jar!/:na]
        at com.alibaba.chaosblade.box.dao.scheduler.BaseSchedulerService.addSchedulerJob(BaseSchedulerService.java:41) ~[chaosblade-box-dao-1.0.1.jar!/:na]
        at com.alibaba.chaosblade.box.dao.scheduler.job.ActivityAsyncCheckSchedulerJob.afterPropertiesSet(ActivityAsyncCheckSchedulerJob.java:62) [chaosblade-box-dao-1.0.1.jar!/:na]
        at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:306) [spring-beans-4.3.13.RELEASE.jar!/:4.3.13.RELEASE]
        at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:230) [spring-beans-4.3.13.RELEASE.jar!/:4.3.13.RELEASE]
        at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:302) [spring-beans-4.3.13.RELEASE.jar!/:4.3.13.RELEASE]
        at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:197) [spring-beans-4.3.13.RELEASE.jar!/:4.3.13.RELEASE]
        at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:543) [spring-context-4.3.13.RELEASE.jar!/:4.3.13.RELEASE]
        at org.springframework.boot.context.embedded.EmbeddedWebApplicationContext.refresh(EmbeddedWebApplicationContext.java:122) [spring-boot-1.5.9.RELEASE.jar!/:1.5.9.RELEASE]
        at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:693) [spring-boot-1.5.9.RELEASE.jar!/:1.5.9.RELEASE]
        at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:360) [spring-boot-1.5.9.RELEASE.jar!/:1.5.9.RELEASE]
        at org.springframework.boot.SpringApplication.run(SpringApplication.java:303) [spring-boot-1.5.9.RELEASE.jar!/:1.5.9.RELEASE]
        at org.springframework.boot.SpringApplication.run(SpringApplication.java:1118) [spring-boot-1.5.9.RELEASE.jar!/:1.5.9.RELEASE]
        at org.springframework.boot.SpringApplication.run(SpringApplication.java:1107) [spring-boot-1.5.9.RELEASE.jar!/:1.5.9.RELEASE]
        at com.alibaba.chaosblade.box.Application.main(Application.java:12) [classes!/:na]
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:1.8.0_161]
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[na:1.8.0_161]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:1.8.0_161]
        at java.lang.reflect.Method.invoke(Method.java:498) ~[na:1.8.0_161]
        at org.springframework.boot.loader.MainMethodRunner.run(MainMethodRunner.java:49) [chaosblade-box-1.0.1.jar:na]
        at com.zaxxer.hikari.pool.ProxyPreparedStatement.executeUpdate(ProxyPreparedStatement.java:61) ~[HikariCP-2.5.1.jar!/:na]
        at com.zaxxer.hikari.pool.HikariProxyPreparedStatement.executeUpdate(HikariProxyPreparedStatement.java) ~[HikariCP-2.5.1.jar!/:na]
        at org.quartz.impl.jdbcjobstore.SimpleTriggerPersistenceDelegate.deleteExtendedTriggerProperties(SimpleTriggerPersistenceDelegate.java:58) ~[quartz-2.3.0.jar!/:na]
        at org.quartz.impl.jdbcjobstore.StdJDBCDelegate.deleteTriggerExtension(StdJDBCDelegate.java:1577) ~[quartz-2.3.0.jar!/:na]
        at org.quartz.impl.jdbcjobstore.StdJDBCDelegate.deleteTrigger(StdJDBCDelegate.java:1561) ~[quartz-2.3.0.jar!/:na]
        at org.quartz.impl.jdbcjobstore.JobStoreSupport.deleteTriggerAndChildren(JobStoreSupport.java:1367) ~[quartz-2.3.0.jar!/:na]
        at org.quartz.impl.jdbcjobstore.JobStoreSupport.removeTrigger(JobStoreSupport.java:1446) ~[quartz-2.3.0.jar!/:na]
        ... 35 common frames omitted

**Comment by xcaspar** [comment]
cc @homily707

**Comment by homily707** [comment]
please assgin this issue to me

**Comment by Jovons** [comment]
Why this is closed? version 1.0.2, still fall into this error.
```
2022-10-18 14:33:53.378 ERROR 8208 --- [           main] c.a.c.b.d.s.BaseSchedulerService         : add scheduler failed,jobId:1582258648193859585

org.quartz.JobPersistenceException: Couldn't check for existence of job: Table 'chaosblade.t_chaos_m_quartz_TRIGGERS' doesn't exist
        at org.quartz.impl.jdbcjobstore.JobStoreSupport.checkExists(JobStoreSupport.java:1990) ~[quartz-2.3.0.jar!/:na]
        at org.quartz.impl.jdbcjobstore.JobStoreSupport$23.execute(JobStoreSupport.java:1981) ~[quartz-2.3.0.jar!/:na]
        at org.quartz.impl.jdbcjobstore.JobStoreCMT.executeInLock(JobStoreCMT.java:245) ~[quartz-2.3.0.jar!/:na]
        at org.quartz.impl.jdbcjobstore.JobStoreSupport.executeWithoutLock(JobStoreSupport.java:3785) ~[quartz-2.3.0.jar!/:na]
        at org.quartz.impl.jdbcjobstore.JobStoreSupport.checkExists(JobStoreSupport.java:1978) ~[quartz-2.3.0.jar!/:na]

```
